mapie/classification.py

```diff
@@ -565,7 +565,7 @@ def _fit_and_predict_oof_model(
 
     def _get_true_label_cumsum_proba(
         self,
-        y: NDArray,
+        y: ArrayLike,
```

Comment:
Is this change justified?

Reply:
yes, otherwise the typing check failes

Reply:
as y is directly the output of the model it may be more complexe than just an array